typing/subst.mli

```diff
@@ -29,22 +28,36 @@ type t
    Indeed, non-variable node of a type are duplicated, with their
    levels set to generic level.  That way, the resulting type is
    well-formed (decreasing levels), even if the original one was not.
-*)
 
-val identity: t
+   In the presence of local substitutions for module types, a substitution for a
+   type expression may fail to produce a well-formed type. In order to confine
+   this issue to local substitution, the type of substitutions is split into a
+   standard and local variant. Only local substitution may expand a module type
+   path into a generic module type. *)
+
+(** Type familly for substitutions *)
+type 'a s
```

Comment:
Naming suggestion: `'a subs`. `t` is a standard short name, but `s` is moderately confusing for no excellent reason.